src/custom-lists/components/overview/sidebar/list-item.tsx

```diff
@@ -30,12 +31,34 @@ class PageList extends Component<Props, State> {
 
     componentDidMount() {
         this.attachEventListeners()
+        this.makeResizableDiv('#teste')
     }
 
     componentWillUnmount() {
         this.removeEventListeners()
     }
 
+    resize = event => {
+        const element = document.querySelector('#teste')
+
+        element.style.width =
+            event.pageX - element.getBoundingClientRect().left + 'px'
+    }
+
+    stopResize = () => {
+        window.removeEventListener('mousemove', this.resize)
+    }
+
+    makeResizableDiv = div => {
+        const resizers = document.querySelectorAll(div + ' .resizable')
```

Comment:
Looking more at these methods, they don't seem to be related to this `<PageList>` component in any way. They just grab the elements matching `#teste .resizable` and do some stuff to them. It would be nice to make a new component to contain this logic + render the needed divs (regardless of whether it's written in React or DOM API code for now).

Reply:
Yeah, for now I'm just testing using pure Javascript. When my test were finished I'll pass every logic to correct place. I'm commit this tests just to keep you all updated.